strr-web/pages/registry-dashboard.vue

```diff
@@ -207,7 +227,7 @@ const updateTableRows = async () => {
   const applications = await getPaginatedApplications(paginationObject)
   if (applications) {
     totalResults.value = applications.total
-    tableRows.value = registrationsToTableRows(applications)
+    tableRows.value = await registrationsToTableRows(applications)
```

Comment:
No `await` is needed here, the `registrationsToTableRows()` is not an async function.